### PR TITLE
feat: Add query all

### PR DESCRIPTION
**Pull request by ptbrowne**
The goal is in Banks to remove all this utils. This will help our services
to move to cozy-client.

### CODE REVIEW

**Comment by kosssi** [comment]
can you add a test?

File: packages/cozy-doctypes/src/Document.js [hunk 1]
```diff
@@ -284,6 +284,50 @@ class Document {
...
+   * * `index` (object): (optional) the query selector index. If not defined, the function will
+   * create it's own index with the keys specified in the selector
+   *
+   *
+   * ```javascript
+   * const documents = await queryAll(cozyClient, 'io.cozy.bills', {vendor: 'Direct Energie'})
+   * ```
+   *
+   */
+  static async queryAll(selector, index) {
```
Comment (Crash--) [inline packages/cozy-doctypes/src/Document.js]: In the doc we have 3 parameters but the function takes only 2 args

Comment (ptbrowne) [reply packages/cozy-doctypes/src/Document.js]: Thanks !

**Comment by ptbrowne** [comment]
OK ?

File: packages/cozy-doctypes/src/Document.js [hunk 2]
```diff
@@ -284,6 +284,49 @@ class Document {
...
+   *
+   * Parameters:
+   *
+   * * `selector` (object): the mango query selector
+   * * `index` (object): (optional) the query selector index. If not defined, the function will
+   * create it's own index with the keys specified in the selector
+   *
+   *
+   * ```javascript
+   * const documents = await queryAll(cozyClient, 'io.cozy.bills', {vendor: 'Direct Energie'})
```
Comment (Crash--) [inline packages/cozy-doctypes/src/Document.js]: are you sure about this usage? (doctype ? cozyClient) ?

Comment (ptbrowne) [reply packages/cozy-doctypes/src/Document.js]: fixed, thanks

**Comment by ptbrowne** [comment]
Added tests, and fixed Quentin remark.

**Pull request merged**